firestore.rules

```diff
@@ -43,6 +43,13 @@ service cloud.firestore {
       allow update: if false;
       allow delete: if false;
     }
+
+    match /platform/stats {
+      allow create: if false;
+      allow get: if false;
```

Comment:
I think for `get` it should be for everyone to read it. Also missing list, which I think we should not allow for the moment.

```suggestion
      allow get: if true;
      allow list: if false;
```